tools/collect_smileys.php

```diff
@@ -67,7 +67,6 @@ function collect_smileys($file) {
 	table.smiley_table td { font: 10pt Courier,monospace,mono; white-space: pre; }
 	table.smiley_table tbody th { text-align: center; }	
 --></style>
-<base href="<?php print "http://" . $_SERVER['SERVER_NAME'] . dirname(dirname($_SERVER['REQUEST_URI'])); ?>" />
```

Comment:
Seems like github doesn't auto expand the code.
https://github.com/djmattyg007/nbbc/blob/820e64739b6d6923f160d0201384bd0663889e88/tools/collect_smileys.php#L29